core/lib/libtomcrypt/src/mpa_desc.c

```diff
@@ -604,7 +604,7 @@ static int isprime(void *a, int b, int *c)
 	return CRYPT_OK;
 }
 
-static int rand(void *a, int size)
+static int myrand(void *a, int size)
```

Comment:
How about mpa_rand? Makes it a bit unique, but still tied to the MPA library?

Reply:
OK, I'll fix.